docs/run_test/platform.md

```diff
@@ -0,0 +1,163 @@
+# Run LISA
+
+- [Quick start](#quick-start)
+  - [Run in Azure](#run-in-azure)
+  - [Run in Ready computers](#run-in-ready-computers)
+- [Run Microsoft tests](#run-microsoft-tests)
+- [Run legacy LISAv2 test cases](#run-legacy-lisav2-test-cases)
+
+## Quick start
+
+`lisa.sh` (for Linux) and `lisa.cmd` (for Windows) are provided to wrap `Poetry`
+for you to run LISA test.
+
+In Linux, you could create an alias for this simple script. For example, add
+below line to add to `.bashrc`:
+
+```bash
+alias lisa="./lisa.sh"
+```
+
+If no argument specified, LISA will run some sample test cases with the default
+runbook (`examples/runbook/hello_world.yml`) on your local computer. You can use
+this way to verify your local LISA environment setup. This test will not modify
```

Comment:
It sounds many "you/your" in those sentences. It needs to unify the style, either to third person or second person, both fine to me. If it uses second person, it also needs to reduced unnecessary "you/your". For example, sentences can be,
"You can use this way to verify the local LISA environment setup. This test will not modify the computer."